Jenkinsfile

```diff
@@ -466,12 +466,6 @@ pipeline {
                                    scons_args: sconsFaultsArgs()
                     }
                     post {
-                        always {
-                            recordIssues enabledForFailure: true,
-                                         aggregatingResults: true,
-                                         tool: gcc4(pattern: 'centos7-gcc-build.log',
-                                                    id: "analysis-gcc-centos7")
-                        }
```

Comment:
We don't want to record issues on this and the other build stages below where this was removed?